docs/input/docs/contributing/how-to-build.md

```diff
@@ -0,0 +1,18 @@
+---
+Order: 20
+Title: How to build Cake.Recipe
+Description: Instructions how to build Cake.Recipe.
+---
+To build this we are using Cake.
+
+On Windows PowerShell run:
+
+```powershell
+./build.ps1
+```
+
+On OSX/Linux run:
+
+```bash
+./build.sh
+```
```

Comment:
I would recommend having a note mentioning not to run the `setup.cake` script directly, as we do some additional changes in the bootstrappers to make the dogfooding work as expected.